Makefile

```diff
@@ -437,3 +437,6 @@ check_diff:
 
 check_shebang:
 	./tools/check-shebang.sh
+
+act:
```

Comment:
Maybe install the tool in the Dockerfile (and in install_prerequisites) and in `make act` just run it instead of installing it?

I think it would be more in line with the other tests in the Makefile.

Reply:
act will start docker from these ci.yml, so it needs to be installed on host.

Reply:
I see, thanks for the clarification